package.json

```diff
@@ -40,7 +40,7 @@
     "coveralls": "^2.11.4",
     "eslint": "^3.0.1",
     "istanbul": "^0.4.2",
-    "jscs": "^3.0.7",
+    "jscs": "^2.11.0",
```

Comment:
Make sure your global version of jscs is updated to 3.x. That's what fixed it for me/